scripts/globals/job_utils/rune_fencer.lua

```diff
@@ -228,6 +245,48 @@ local function getAnimationSwipeLunge(weaponSkillType) -- verified via retail ac
     return weaponAnimationMap[weaponSkillType]
 end
 
+local function getAnimationGambit(weaponSkillType) -- verified via retail action packets exclusively
+
+    local weaponAnimationMap =
+    {
+        [xi.skill.NONE]         = 16,
+        [xi.skill.HAND_TO_HAND] = 16,
+        [xi.skill.DAGGER]       = 17,
+        [xi.skill.SWORD]        = 15,
+        [xi.skill.GREAT_SWORD]  = 20,
+        [xi.skill.AXE]          = 18, -- club/axe share the same animation for Gambit
+        [xi.skill.GREAT_AXE]    = 19, -- gaxe/scythe share the animation for Gambit
+        [xi.skill.SCYTHE]       = 19,
+        [xi.skill.POLEARM]      = 21,
+        [xi.skill.KATANA]       = 22,
+        [xi.skill.GREAT_KATANA] = 23,
+        [xi.skill.CLUB]         = 18,
+        [xi.skill.STAFF]        = 24,
+    }
+    return weaponAnimationMap[weaponSkillType]
+end
+
+local function getAnimationRayke(weaponSkillType) -- verified via retail action packets exclusively
+
+    local weaponAnimationMap =
+    {
+        [xi.skill.NONE]         = 26,
+        [xi.skill.HAND_TO_HAND] = 26,
+        [xi.skill.DAGGER]       = 27,
+        [xi.skill.SWORD]        = 25,
+        [xi.skill.GREAT_SWORD]  = 30,
+        [xi.skill.AXE]          = 28, -- club/axe share the same animation for Rayke
+        [xi.skill.GREAT_AXE]    = 29, -- gaxe/scythe share the animation for Rayke
+        [xi.skill.SCYTHE]       = 29,
+        [xi.skill.POLEARM]      = 31,
+        [xi.skill.KATANA]       = 32,
+        [xi.skill.GREAT_KATANA] = 33,
+        [xi.skill.CLUB]         = 28,
+        [xi.skill.STAFF]        = 34,
+    }
+    return weaponAnimationMap[weaponSkillType]
+end
```

Comment:
I notice that these three maps are all the same, with increments of 10 added to them, so you could define this as a single table outside the scope of these functions, and then have the functions return `table[weaponSkillType + specificOffset]`